src/main/js/dynamic/loaded/users/PermissionTable.js

```diff
@@ -47,7 +48,15 @@ class PermissionTable extends Component {
         const updatedValue = type === 'checkbox' ? checked.toString()
             .toLowerCase() === 'true' : value;
         const trimmedValue = (Array.isArray(updatedValue) && updatedValue.length > 0) ? updatedValue[0] : updatedValue;
-        const newPermissions = Object.assign(permissionsData, { [name]: trimmedValue });
+
+        let newPermissions = Object.assign(permissionsData, { [name]: trimmedValue });
+        if (newPermissions && newPermissions.descriptorName !== 'Settings') {
+            newPermissions = Object.assign(newPermissions, {
+                [PERMISSIONS_TABLE.UPLOAD_READ]: undefined,
+                [PERMISSIONS_TABLE.UPLOAD_WRITE]: undefined,
+                [PERMISSIONS_TABLE.UPLOAD_DELETE]: undefined
+            });
+        }
```

Comment:
If the settings permissions happen to be set, unset them if the descriptor isn't 'Settings'.

Reply:
SAML configuration is in Authentication not Settings.  I think you need to change the keyword from Settings to 'Authentication'

Reply:
Oof, good catch.